specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/applicationGateway.json

```diff
@@ -801,13 +801,13 @@
           "$ref": "#/definitions/ApplicationGatewayProbeHealthResponseMatch",
           "description": "Criterion for classifying a healthy probe response."
         },
-        "backendPoolName": {
-          "type": "string",
-          "description": "Name of backend pool of application gateway to which probe request will be sent."
+        "backendAddressPool": {
```

Comment:
this is a breaking change in stable version, pls create a new version

Reply:
@yungezz , this should not be breaking for any customer. Reason is that previously, these changes got merged incorrectly before the NRP API changes were checked in - so no customer can use it. The NRP API (the only version) has gone in NRP/124. So, we need to correct the definitions here as per the actual backend API.

Reply:
Are you saying the properties in the repo (backendPoolName...) were never supported? This change is to align the documentation/sdk with what the backend supports.

Or are you saying you've changed your backend and need to update the documentation/sdk to represent this change?

Reply:
The first option. backendPoolName.. and backendHttpSettingName were never supported. This change is to align the documentation/sdk with what the backend supports.